low-level/sensors/impl/generic/node_sas_port.py

```diff
@@ -181,47 +184,141 @@ def initialize(self, conf_reader, msgQlist, product):
 
         return True
 
-    def check_and_send_alert(self, new_phy_link_count):
+    def sort_phy_list(self, phy_list):
```

Comment:
Directly sorting the phy names list does not work for us because it places "phy-0:10" before "phy-0:2" ... So we have to extract the phy numbers and sort according to them

Reply:
Seems fine.